integrationTests/interface.go

```diff
@@ -54,16 +54,16 @@ type NetworkShardingUpdater interface {
 
 // Facade is the node facade used to decouple the node implementation with the web server. Used in integration tests
 type Facade interface {
-	GetBalance(address string) (*big.Int, error)
-	GetUsername(address string) (string, error)
-	GetValueForKey(address string, key string) (string, error)
-	GetAccount(address string) (dataApi.AccountResponse, error)
-	GetESDTData(address string, key string, nonce uint64) (*esdt.ESDigitalToken, error)
-	GetNFTTokenIDsRegisteredByAddress(address string) ([]string, error)
-	GetESDTsWithRole(address string, role string) ([]string, error)
-	GetAllESDTTokens(address string) (map[string]*esdt.ESDigitalToken, error)
-	GetESDTsRoles(address string) (map[string][]string, error)
-	GetKeyValuePairs(address string) (map[string]string, error)
+	GetBalance(address string, options api.AccountQueryOptions) (*big.Int, api.BlockInfo, error)
```

Comment:
the problem I have with this DTO struct usage is the fact that we can have conflicting values on it. Such as:
```
a := AccountQueryOptions{
    OnFinalBlock:   true,
    OnStartOfEpoch: 4,
}
```
Which will always make us go the part of the code that handles this struct

Reply:
yup